resources/timezones/zones.json

```diff
@@ -486,7 +486,7 @@
     },
     "Africa/Casablanca": {
       "ics": [
-        "BEGIN:STANDARD\r\nTZOFFSETFROM:+0100\r\nTZOFFSETTO:+0100\r\nTZNAME:+01\r\nDTSTART:19700101T000000\r\nEND:STANDARD"
+        "BEGIN:STANDARD\r\nTZOFFSETFROM:+0100\r\nTZOFFSETTO:+0100\r\nTZNAME:%z\r\nDTSTART:19700101T000000\r\nEND:STANDARD"
```

Comment:
This ain't looking right.

Reply:
But event before, `+01` doesn't seem right too. It should be a string like EST.

Ref https://www.kanzaki.com/docs/ical/tzname.html